plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestingH2JdbcClient.java

```diff
@@ -162,7 +165,7 @@ public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connect
             case Types.BIGINT:
                 return Optional.of(bigintColumnMapping());
 
-            case Types.REAL:
+            case Types.REAL, Types.FLOAT:
```

Comment:
why both?
did you mean to remove `Types.REAL` case?

Reply:
not exactly, we've been creating REAL as "float" column, that's why on a read path we've needed float as well. I've changed a write function to use real type instead